Code/Tools/AssetProcessor/native/ui/MainWindow.cpp

```diff
@@ -386,7 +386,7 @@ void MainWindow::Activate()
         this, writeJobFilterSettings);
 
     // Asset view
-    m_sourceAssetTreeFilterModel = new AssetProcessor::SourceAssetTreeFilterModel(this);
+    m_sourceAssetTreeFilterModel = new AssetProcessor::AssetTreeFilterModel(this);
```

Comment:
if the class name is being renamed (SourceAssetTreeFilterModel -> AssetTreeFilterModel ) why not the data member "m_sourceAssetTreeFilterModel" ?

Reply:
There's actually 3 of these, the other 2 are below: 
`m_intermediateAssetTreeFilterModel = new AssetProcessor::AssetTreeFilterModel(this);` and `m_productAssetTreeFilterModel = new AssetProcessor::AssetTreeFilterModel(this);`

SourceAssetTreeFilterModel is actually the most accurate name for this given the other 2 variables. The class I removed was likely named to match this variable rather than the other way around.